subsys/nrf_security/src/drivers/cracen/cracenpsa/src/kmu.c

```diff
@@ -840,6 +842,20 @@ psa_status_t cracen_kmu_get_key_slot(mbedtls_svc_key_id_t key_id, psa_key_lifeti
 	return PSA_SUCCESS;
 }
 
+psa_status_t cracen_kmu_block(const psa_key_attributes_t *key_attr)
```

Comment:
I wonder wheter we can have (paralel) API `psa_status_t cracen_kmu_block_raw(const size_t slot_id, const size_t slot_count)`
rationale: nowadays we are using key-ids as key-slot-number. We can avoid of construction key_attr then.

Reply:
Sure thing: https://github.com/nrfconnect/sdk-nrf/compare/d34db6c7f4aa80e516f19245190fd1a55e9a37fe..a26c5537b1373d70b030dfb9a3d2a7b41ef4872c